convert-hf-to-gguf.py

```diff
@@ -1301,15 +1301,23 @@ def set_vocab(self):
         try:
             self. _set_vocab_sentencepiece()
         except FileNotFoundError:
-            self._set_vocab_llama_hf()
-
-        special_vocab = gguf.SpecialVocab(self.dir_model, load_merges=False,
-                                          special_token_types = ['prefix', 'suffix', 'middle', 'eot'])
-        special_vocab._set_special_token("prefix", 32007)
-        special_vocab._set_special_token("suffix", 32008)
-        special_vocab._set_special_token("middle", 32009)
-        special_vocab._set_special_token("eot",    32010)
-        special_vocab.add_to_gguf(self.gguf_writer)
+            try:
+                self._set_vocab_llama_hf()
+            except TypeError:
```

Comment:
```suggestion
            except (FileNotFoundError, TypeError):
```

Reply:
Should also include `FileNotFoundError` exception as I and @ggerganov are noticing the same issue even after using this PR.